win32ss/user/ntuser/main.c

```diff
@@ -654,11 +654,7 @@ InitThreadCallback(PETHREAD Thread)
     /* Create the default input context */
     if (IS_IMM_MODE())
     {
-        PIMC pIMC = UserCreateInputContext(0);
-        if (pIMC)
-        {
-            UserDereferenceObject(pIMC);
-        }
+        (VOID)UserCreateInputContext(0);
```

Comment:
A comment telling that the context will automatically go away at thread termination time would be nice :)
(so that we don't get the impression here, by reading the code, that you're leaking the returned input context).